net/softethervpn/patches/100-gcc14.patch

```diff
@@ -0,0 +1,121 @@
+--- a/src/Mayaqua/Secure.c
```

Comment:
Where does this patch come from? I couldnt see it in the upstream repository for the stable version.

Reply:
Upstream probably has this fixed. It’s just a real pain to update this package as the patches use CRLF endings.

Reply:
Well, as we do not have man power to maintain this package and based on your statement, it seems like maintainership burden, why do we have this package in our repository? Drop it and it can be added back, when there is someone, who can take care of it. We do have a plenty of packages here and with just a few maintainers, it is not possible to have everything up to date and users can use softethervpn5.

Reply:
Maybe @fededim should comment.